api_ui/api.yaml

```diff
@@ -1027,7 +1060,7 @@ components:
         - qrCodeLink
         - schemaType
       properties:
-        qrCodeLink:
+        qrCodeLink: # TODO: Rename to QRCode. It can be a Link or a Raw QR
```

Comment:
this TODO here is correct?

Reply:
I will remove. Ideally we should change it because we always return a QR code, no matter if it is a link or a raw qr. But renaming it would be a breaking change for ui.